tmt/steps/discover/__init__.py

```diff
@@ -234,9 +244,9 @@ def discover(context, **kwargs):
         return discover
 
     @classmethod
-    def options(cls, how=None):
+    def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecoratorType]:
         """ Prepare command line options for given method """
-        return [
+        return cast(List[tmt.options.ClickOptionDecoratorType], [
```

Comment:
@janhavlin as discussed, this needs a change

Reply:
Right, fixed.